presets/4.3/tune/fpvian_basher_freestyle.txt

```diff
@@ -0,0 +1,181 @@
+#$ TITLE: Basher Freestyle 5" by FPVian
+#$ FIRMWARE_VERSION: 4.3
+#$ CATEGORY: TUNE
+#$ STATUS: COMMUNITY
```

Comment:
we decided all the new presets become experimental first.
Then, when 3-4 pilots try them with different options from the actual Configurator tab, you can make a PR to make it community if you want.

Reply:
Changed to experimental. How do pilots submit that they've tried the preset and it works? Comment on this PR?

Reply:
Yes that's why you include the PR link in the discussion tag